components/serverless/docker-compose.serverless.yml

```diff
@@ -1,7 +1,7 @@
 services:
   nuclio:
     container_name: nuclio
-    image: quay.io/nuclio/dashboard:1.11.24-amd64
+    image: quay.io/nuclio/dashboard:1.13.0-amd64
```

Comment:
Please add a changelog entry.

Also, you still need to apply an equivalent change to the Helm chart.